docs/docs/reference/dropped-features/wildcard-init.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: doc-page
+title: "Dropped: wildcard initializer"
+---
+
+The syntax
+```scala
+  var x: A = _
+```
+that was used to indicate an uninitialized field, has been dropped.
+At its place there is a special value `uninitialized` in the `scala.compiletime` package. To get an uninitialized field, you now write
+```scala
+import scala.compiletime.uninitialized
+
+var x: A = uninitialized
```

Comment:
Currently, we also support

```scala
  var a = uninitialized[A]
```

This seems to be leaking implementation details.

Reply:
Right. We it's better to define it like this:
```scala
@compileTimeOnly("`uninitialized` can only be used as the right hand side of a mutable field definition")
  def uninitialized: Nothing = ???
```